lib/features/family/features/parent_giving_flow/presentation/pages/parent_amount_page.dart

```diff
@@ -34,6 +33,8 @@ class ParentAmountPage extends StatefulWidget {
 
 class _ParentAmountPageState extends State<ParentAmountPage> {
   final initialamount = 25;
+  bool isActive = false;
```

Comment:
Can you put a comment here to explain what "isActive" means? Now I can only "infer" it from the code but not sure about its intention.

Reply:
Renamed to: hasOverwrittenPlaceholderValue